hops/serving.py

```diff
@@ -784,6 +784,7 @@ def __init__(self, serving_json):
         self.id = serving_json[constants.REST_CONFIG.JSON_SERVING_ID]
         self.name = serving_json[constants.REST_CONFIG.JSON_SERVING_NAME]
         self.model_path = serving_json[constants.REST_CONFIG.JSON_SERVING_MODEL_PATH]
+        self.model_name = serving_json[constants.REST_CONFIG.JSON_SERVING_MODEL_NAME]
```

Comment:
Can you add the `get_model_name` method?

Reply:
Added function